src/realm/geospatial.hpp

```diff
@@ -117,6 +117,7 @@ struct GeoBox {
 //   - Any interior ring must be entirely contained by the outer ring.
 //   - Interior rings cannot intersect or overlap each other. Interior rings cannot share an edge.
 struct GeoPolygon {
+    GeoPolygon() {}
```

Comment:
```suggestion
    GeoPolygon() = default;
```

There is a very subtle difference here. In this case it doesn't matter much, but it is always better to prefer `= default;` over `{}` where possible.